<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>320</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Recept kereső</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>471</width>
      <height>311</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="Alapanyagok">
     <attribute name="title">
      <string>Alapanyagok</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>71</width>
        <height>131</height>
       </rect>
      </property>
      <property name="title">
       <string>Húsok</string>
      </property>
      <widget class="QRadioButton" name="marha">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>82</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Marha</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="csirke">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>40</y>
         <width>82</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Csirke</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="pulyka">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>60</y>
         <width>82</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Pulyka</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="hal">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>80</y>
         <width>82</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Hal</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="sertes">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>100</y>
         <width>82</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Sertés</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>10</y>
        <width>361</width>
        <height>241</height>
       </rect>
      </property>
      <property name="title">
       <string>Zöldségek - Gyümölcs - Fűszer</string>
      </property>
      <widget class="Line" name="line">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>120</y>
         <width>361</width>
         <height>16</height>
        </rect>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
      <widget class="QCheckBox" name="checkBox">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>70</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Hagyma</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="checkBox_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>40</y>
         <width>70</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Krumpli</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="checkBox_3">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>60</y>
         <width>70</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Batáta</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="checkBox_4">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>80</y>
         <width>81</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Paradicsom</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="checkBox_5">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>100</y>
         <width>70</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Zeller</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="checkBox_6">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>20</y>
         <width>70</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Répa</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="checkBox_7">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>40</y>
         <width>70</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Karalábé</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="checkBox_8">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>60</y>
         <width>70</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Saláta</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="checkBox_9">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>80</y>
         <width>70</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Káposzta</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="checkBox_10">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>100</y>
         <width>70</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Cukkini</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="checkBox_11">
       <property name="geometry">
        <rect>
         <x>190</x>
         <y>20</y>
         <width>70</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Padlizsán</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="checkBox_12">
       <property name="geometry">
        <rect>
         <x>190</x>
         <y>40</y>
         <width>70</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Uborka</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="checkBox_13">
       <property name="geometry">
        <rect>
         <x>190</x>
         <y>60</y>
         <width>70</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Paprika</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="checkBox_14">
       <property name="geometry">
        <rect>
         <x>190</x>
         <y>80</y>
         <width>70</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Karfiol</string>
       </property>
      </widget>
      <widget class="QFrame" name="frame">
       <property name="geometry">
        <rect>
         <x>260</x>
         <y>10</y>
         <width>91</width>
         <height>111</height>
        </rect>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
      </widget>
      <widget class="Line" name="line_2">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>130</y>
         <width>20</width>
         <height>111</height>
        </rect>
       </property>
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
      <widget class="QCheckBox" name="checkBox_16">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>130</y>
         <width>70</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Bazsalikom</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="checkBox_17">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>150</y>
         <width>91</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Petrezselyem</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="checkBox_18">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>170</y>
         <width>70</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Oregánó</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="checkBox_19">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>190</y>
         <width>70</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Fahéj</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="checkBox_20">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>210</y>
         <width>70</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Dió</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="checkBox_21">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>130</y>
         <width>81</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Szerecsendió</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="checkBox_22">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>150</y>
         <width>70</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Chili</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="checkBox_23">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>170</y>
         <width>70</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Kakukkfű</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="checkBox_24">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>190</y>
         <width>70</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Koriander</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="checkBox_25">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>210</y>
         <width>70</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Tárkony</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="checkBox_26">
       <property name="geometry">
        <rect>
         <x>190</x>
         <y>130</y>
         <width>70</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Alma</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="checkBox_27">
       <property name="geometry">
        <rect>
         <x>190</x>
         <y>150</y>
         <width>70</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Narancs</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="checkBox_28">
       <property name="geometry">
        <rect>
         <x>190</x>
         <y>170</y>
         <width>70</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Körte</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="checkBox_29">
       <property name="geometry">
        <rect>
         <x>190</x>
         <y>190</y>
         <width>70</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Citrom</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="checkBox_30">
       <property name="geometry">
        <rect>
         <x>190</x>
         <y>210</y>
         <width>70</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Ananász</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="checkBox_31">
       <property name="geometry">
        <rect>
         <x>270</x>
         <y>130</y>
         <width>70</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Meggy</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="checkBox_32">
       <property name="geometry">
        <rect>
         <x>270</x>
         <y>150</y>
         <width>81</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Cseresznye</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="checkBox_33">
       <property name="geometry">
        <rect>
         <x>270</x>
         <y>170</y>
         <width>70</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Lime</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="checkBox_34">
       <property name="geometry">
        <rect>
         <x>270</x>
         <y>190</y>
         <width>70</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Szilva</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="checkBox_35">
       <property name="geometry">
        <rect>
         <x>270</x>
         <y>210</y>
         <width>70</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Barack</string>
       </property>
      </widget>
     </widget>
     <widget class="QPushButton" name="recept_keres">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>260</y>
        <width>311</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Recept keresése</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Recept</string>
     </attribute>
     <widget class="QPushButton" name="recept_mentes">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>260</y>
        <width>231</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Recept mentése</string>
      </property>
     </widget>
     <widget class="QTextBrowser" name="recept_out">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>30</y>
        <width>471</width>
        <height>231</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 11pt &quot;MS Shell Dlg 2&quot;;</string>
      </property>
     </widget>
     <widget class="QLabel" name="receptcime">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>5</y>
        <width>461</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 75 12pt &quot;MS Shell Dlg 2&quot;;</string>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Tápanyagok</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>Mentett receptek</string>
     </attribute>
     <widget class="QListView" name="listView">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>471</width>
        <height>291</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
